db/background_mgr_resync_dcp_test.go

```diff
@@ -336,6 +336,43 @@ func TestResyncManagerDCPRunTwice(t *testing.T) {
 	wg.Wait()
 }
 
+func TestResyncImportComputeStat(t *testing.T) {
+	if base.UnitTestUrlIsWalrus() {
+		t.Skip("Test requires Couchbase Server")
+	}
+	const docsToCreate = 1000
+	db, ctx := setupTestDBForResyncWithDocs(t, docsToCreate, false)
+	defer db.Close(ctx)
+
+	resycMgr := NewResyncManagerDCP(db.MetadataStore, base.TestUseXattrs(), db.MetadataKeys)
+	require.NotNil(t, resycMgr)
+	db.ResyncManager = resycMgr
+
+	options := map[string]interface{}{
+		"database":            db,
+		"regenerateSequences": false,
+		"collections":         ResyncCollections{},
```

Comment:
does this test work with 

matrix of:

SG_TEST_USE_GSI=true,false
SG_TEST_USE_XATTRS=true,false
SG_TEST_USE_DEFAULT_COLLECTION=true,false

?